index.html

```diff
@@ -0,0 +1,212 @@
+              the event.
+            </p>
+          </div>
+        </div>
+        <div class="main-program-content flex-center">
+          <div class="main-program-img">
+            <img src="https://img.icons8.com/emoji/48/000000/microphone-emoji.png" alt="presentation" />
+          </div>
+          <div class="main-subject">
+            <p>Presentation</p>
+          </div>
+          <div class="main-content">
+            <p>
+              Host of the event gives a short speech about her journey til date, inspirations, what is about to be
+              experienced and what to come.
+            </p>
+          </div>
+        </div>
+        <div class="main-program-content flex-center">
+          <div class="main-program-img">
+            <img src="https://img.icons8.com/color/50/000000/fashion-model.png" alt="runway model" />
+          </div>
+          <div class="main-subject">
+            <p>Fashion Runway</p>
+          </div>
+          <div class="main-content">
+            <p>
+              Our models showcase our different line and experties and glamour for all weather on the runway.
+            </p>
+          </div>
+        </div>
+        <div class="main-program-content flex-center">
+          <div class="main-program-img">
+            <img src="https://img.icons8.com/color/48/000000/people-working-together.png" alt="get together" />
+          </div>
+          <div class="main-subject">
+            <p>Get Together</p>
+          </div>
+          <div class="main-content">
+            <p>
+              Our guest meet the host and other fashion celebrities to know each other and merry.
+            </p>
+          </div>
+        </div>
+        <div class="main-program-content flex-center">
+          <div class="main-program-img">
+            <img src="https://img.icons8.com/dusk/64/000000/donate.png" alt="donate" />
+          </div>
+          <div class="main-subject">
+            <p>Fund-raiser</p>
+          </div>
+          <div class="main-content">
+            <p>
+              The platform is willing to help young and talented fashion designer, to showcase what they can do
+              on a large scale.
+            </p>
+          </div>
+        </div>
+      </div>
+      <div class="see-more">
+        <a href="#">SEE MORE</a>
+      </div>
+    </div>
+
+    <div class="get-ticket-btn flex-center">
+      <button type="submit">Get Tickets</button>
+    </div>
+  </section>
+
+  <section class="feature">
+    <div class="feature-speaker">
+      <div class="flex flex-center">
+        <p class="feature-title">Celebrity Guests</p>
+        <div class="feature-title-line"></div>
+      </div>
+      <div class="speaker-grid">
+
+
+      </div>
+
+      <div class="show-more-btn">
+        <button type="submit">
+          MORE <i class="fa fa-chevron-circle-down"></i>
+        </button>
+      </div>
```

Comment:
### [OPTIONAL]
- [ ] I would suggest in the mobile version to implement the see more functionality that will be hiding some speakers and show them when the button is clicked.

![image](https://user-images.githubusercontent.com/65068771/147109367-8807de1e-ccc5-4a70-97b6-453d8c962d4b.png)

Reply:
Thanks👌 I'll do that in the subsequent project 😎. Since It's optional.